govc/vm/console.go

```diff
@@ -163,6 +165,13 @@ func (cmd *console) Run(ctx context.Context, f *flag.FlagSet) error {
 		}.Encode()
 
 		link = u.String()
+	} else if cmd.wss {
+		ticket, err := vm.AcquireTicket(ctx, string(types.VirtualMachineTicketTypeWebmks))
```

Comment:
Should this switch maybe be moved to where the "normal" (clone) ticket would otherwise be acquired? So we don't always acquire both if we only need the Webmks.

Reply:
Yes, good catch, moved the wss block to avoid AcquireCloneTicket